subprojects/common/junit-utils/src/main/java/ru/avito/util/RandomUtils.kt

```diff
@@ -184,6 +186,15 @@ fun randomPhone(): String {
     return "+7" + randomString(10, Characters.Digits)
 }
 
+@Deprecated("start and end could be negative", replaceWith = ReplaceWith("randomDate(start, end)"))
 fun randomDate(start: Long, end: Long): Date {
     return Date(randomLong(start, end))
 }
+
+fun randomDate(start: Date, end: Date): Date {
+    return Date(randomLong(start.time, end.time))
+}
+
+fun randomDate(): Date = Date(randomLong())
```

Comment:
```suggestion
fun randomDate(): Date = Date(randomLong(min = 0))
```
I'd prefer to define this contract explicitly.

Reply:
`randomLong(min = 0, max = Long.MAX_VALUE)`